cypress/e2e/backoffice/sor-contracts.cy.js

```diff
@@ -188,10 +188,7 @@ describe('SOR-Contracts - When Add selected', () => {
     cy.intercept(
       {
         method: 'GET',
-        pathname: '/api/backoffice/contracts',
-        query: {
-          contractorReference: '*',
-        },
```

Comment:
How come query is removed? I'd have expected this object to receive an additional isActive key instead :thinking: